theano/gpuarray/blas.py

```diff
@@ -488,6 +488,11 @@ def flops(self, inp, outp):
     def get_params(self, node):
         return node.inputs[0].type.context
 
+    def get_op_params(self):
+        return [("ADDR32_MAX", "4294967295"),
+                ("SADDR32_MIN", "-2147483648"),
+                ("SADDR32_MAX",  "2147483647")]
```

Comment:
These should probably be embedded in the code rather than be op_params.  I doubt they will ever change.